### ISSUE TITLE
about test AP

### ISSUE THREAD

**Issue by fukexue** [monster-ghost/MonoDistill#2]
I'm sorry to bother you, but I found that the training model you provided could not achieve the results described in the paper after it was submitted to Kitti's official test.
The result as follow:

Benchmark | Easy | Moderate | Hard
Car (Detection) | 93.23 % | 87.73 % | 78.06 %
Car (Orientation) | 92.58 % | 86.44 % | 76.45 %
Car (3D Detection) | 16.75 % | 12.58 % | 10.45 %
Car (Bird's Eye View) | 25.18 % | 18.13 % | 15.41 %

**Comment by monster-ghost** [comment]
The checkpoint we provided is trained on the train set(3K). You can eval it on the validation set.
If you want to submit to KITTI's official test, you'd better train on the trainval dataset(7K).

**Comment by Senwang98** [comment]
@fukexue @monster-ghost 
Hi, I am trying to submit to KITTI too. 
I am curious that the final training version's writelist is `['Car']` or `['Pedestrian', 'Car', 'Cyclist']`?

**Comment by czy341181** [comment]
@Senwang98   It is only Car. If you submit the results of multiple categories, the AP of the vehicle will decrease slightly
